### DISCUSSION TITLE
Proper way to write Display headings?

### DISCUSSION THREAD

**Discussion by stsrki** [twbs/bootstrap#39035] [Q&A] [answered]
In the documentation under the Display headings(https://getbootstrap.com/docs/5.3/content/typography/#display-headings) all elements are written with an `h1` element. Is this the proper way to write it semantically? The problem is that in case that we have multiple Display headings on the same page, and if all of them are made with `h1` then it goes against SEO rules. Only one `h1` should be on a page.

**Comment by nkdas91** [comment, marked as answer]
Hi @stsrki,

As far as I understand, those are just examples to show how we can make a heading (doesn't have to be an `h1`) more prominent / stand out by using `display-*` classes.

_When you need a heading to stand out, consider using a display heading—a larger, slightly more opinionated heading style._

And if you are concerned about Bootstrap's SEO, if you inspect those examples, you can see those are `<div>`s not headings.

**Reply by stsrki** [reply]
Thanks for the response. That makes sense. I was confused because unlike regular heading there wasn't note about using it differently. And I didn't inspect the Display headings in this case.

**Note from the regular headings docs:**

> .h1 through .h6 classes are also available, for when you want to match the font styling of a heading but cannot use the associated HTML element.
